srcpkgs/python-isbnlib/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'python-isbnlib'
+pkgname=python-isbnlib
+version=3.9.6
+revision=1
+archs=noarch
+wrksrc="isbnlib-${version}"
+build_style=python-module
+pycompile_module="isbnlib"
+hostmakedepends="python-setuptools python3-setuptools"
```

Comment:
needs depends="python"

same for python3- subpackage but python3 on depends= instead

Reply:
OK, I'm fixing the others as well